basic-steps/CORE-STEPS.md

```diff
@@ -72,8 +72,28 @@ node {
 though this would not automatically adjust the message according to the status of _previous_ builds as the standard mail notifier does.
 That would be possible only via [JENKINS-26834](https://issues.jenkins-ci.org/browse/JENKINS-26834).
 
+# Build wrappers
+
+The `wrap` step may be used to run a build wrapper defined originally for freestyle projects.
+In a workflow, any block of code (inside `node`) may be wrapped in this way, not necessarily the whole build.
+
+For example, the Xvnc plugin allows a headless build server to run GUI tests by allocating an in-memory-only X11 display.
+To use this plugin from a workflow, assuming a version with the appropriate update:
+
+```groovy
+node('linux') {
+  wrap([$class: 'Xvnc', useXauthority: true]) {
```

Comment:
Thanks for the clarification.
*BTW - in the linked documents the example checkpoint is inside a node :-o 
 Most people will have stopped reading by the get down to the important bit of info that is burried further down.*